Formula/v/vrpn.rb

```diff
@@ -1,13 +1,12 @@
 class Vrpn < Formula
   desc "Virtual reality peripheral network"
   homepage "https://github.com/vrpn/vrpn/wiki"
-  url "https://github.com/vrpn/vrpn/releases/download/version_07.35/vrpn_07.35.zip"
-  sha256 "06b74a40b0fb215d4238148517705d0075235823c0941154d14dd660ba25af19"
+  url "https://github.com/vrpn/vrpn.git",
+    tag:      "v07.36",
+    revision: "79deb000cc0b47ae49a80c92c78167c02d8a04d8"
```

Comment:
Small changes here for 2 spaces 😅
```suggestion
      tag:      "v07.36",
      revision: "79deb000cc0b47ae49a80c92c78167c02d8a04d8"
```